src/main/resources/assets/dumplings_delight/lang/vi_vn.json

```diff
@@ -0,0 +1,56 @@
+    "item.dumplings_delight.tomato_egg_boiled_dumpling": "Sủi cảo nhân cà chua trứng",
+    "item.dumplings_delight.pork_potato_boiled_dumpling": "Sủi cảo nhân thịt heo khoai tây",
+    "item.dumplings_delight.mushroom_boiled_dumpling": "Sủi cảo nhân nấm",
+    "item.dumplings_delight.pork_carrot_wonton": "Hoành thánh nhân thịt heo cà rốt",
+    "item.dumplings_delight.vinegar": "Giấm",
+    "item.dumplings_delight.fungus_boiled_dumpling": "Sủi cảo nhân nấm địa ngục",
+    "item.dumplings_delight.dandelion_leaf_boiled_dumpling": "Sủi cảo nhân lá bồ công anh",
+    "item.dumplings_delight.calamari_boiled_dumpling": "Sủi cảo nhân mực",
+    "item.dumplings_delight.chicken_mushroom_boiled_dumpling": "Sủi cảo nhân thịt gà nấm",
+    "item.dumplings_delight.pork_celery_boiled_dumpling": "Sủi cảo nhân thịt heo cần tây",
+    "item.dumplings_delight.pork_mushroom_wonton": "Hoành thánh nhân thịt heo nấm",
+    "item.dumplings_delight.pork_cabbage_boiled_dumpling": "Sủi cảo nhân thịt heo cải thảo",
+    "item.dumplings_delight.salmon_boiled_dumpling": "Sủi cảo nhân cá hồi",
+    "item.dumplings_delight.pork_cabbage_wonton": "Hoành thánh nhân thịt heo cải thảo",
+    "item.dumplings_delight.cod_boiled_dumpling": "Sủi cảo nhân cá tuyết",
+    "item.dumplings_delight.rabbit_meat_boiled_dumpling": "Sủi cảo nhân thịt thỏ",
+    "item.dumplings_delight.calamari": "Thịt mực",
+    "item.dumplings_delight.chinese_cabbage": "Cải thảo",
+    "item.dumplings_delight.chinese_cabbage_leaf": "Lá cải thảo",
+    "item.dumplings_delight.eggplant": "Cà tím",
+    "item.dumplings_delight.garlic_chive": "Hẹ",
+    "item.dumplings_delight.fennel": "Tiểu hồi",
+    "item.dumplings_delight.pork_fennel_boiled_dumpling": "Sủi cảo nhân thịt heo tiểu hồi",
+    "item.dumplings_delight.garlic_chive_egg_boiled_dumpling": "Sủi cảo nhân trứng trộn hẹ",
+    "block.dumplings_delight.chinese_cabbages": "Hạt cải thảo",
+    "block.dumplings_delight.garlic": "Tỏi",
+    "block.dumplings_delight.greenonion": "Hành lá",
+    "block.dumplings_delight.eggplant": "Hạt cà tím",
+    "block.dumplings_delight.garlic_chive": "Hạt giống hẹ",
+    "block.dumplings_delight.fennel": "Hạt tiểu hồi",
+    "block.dumplings_delight.chinese_cabbage_crate": "Giỏ cải thảo",
+    "block.dumplings_delight.garlic_crate": "Giỏ tỏi",
+    "block.dumplings_delight.greenonion_crate": "Giỏ hành lá",
+    "block.dumplings_delight.eggplant_crate": "Giỏ cà tím",
+    "block.dumplings_delight.garlic_chive_crate": "Giỏ hẹ",
+    "block.dumplings_delight.fennel_crate": "Giỏ tiểu hồi",
+    "block.dumplings_delight.dumpling_medley": "Đĩa sủi cảo",
+    "dumplings_delight.advancement.root": "Dumpling's Delight",
+    "dumplings_delight.advancement.root.desc": "Một ai đó đã từng nói, \"Hương vị không quan trọng, miễn nó là sủi cảo là được! \"",
+    "dumplings_delight.advancement.consume_dumpling.desc": "Ăn bất kì loại sủi cảo nào",
+    "dumplings_delight.advancement.consume_dumpling": "It's time to ăn sủi cảo!",
```

Comment:
is this intentionally partly in english?

Reply:
Yes, this isn't a mistake, i can translate it fully in Vietnamese but it will be boring in achievements

Reply:
And root line is the mod's title so i don't translate that too

Reply:
ah I was just referring to the consume dumpling advancement title

thanks! will update the mod soon